fission-cli/library/Fission/CLI/Release.hs

```diff
@@ -33,7 +33,8 @@ checkLatestRelease ::
 checkLatestRelease = do
   attempt Env.get >>= \case
     Left  err ->
-      CLI.Error.put err "Unable to parse config"
+      -- Also happens when the CLI isn't setup yet
+      logDebug @Text "Unable to parse config"
```

Comment:
We should also re-surface the error so that the caller can be alerted that this jappened

```haskell
Left err -> do
  logDebug @Text "Unable to parse config"
  return $ Left err
```

Reply:
(so that they can react and let the user know contextually)

Reply:
Ah, yeah right. It *does* swallow the errors! I was thinking it didn't do that :sweat_smile: The types misguided me, but yeah, we talked about that and `rescue`.

Reply:
[Discussed in Discord, but for those playing along at home]

Yeah, it's clearer in 0.3.x

```haskell
attempt :: innerM a -> outerM (Either (ErrorCase innerM) a) 
```

...where

Reply:
> I'm pretty sure it's intentional that the errors get swallowed. Otherwise the dispatch function doesn't continue with it's job.

👀

Reply:
Hmm maybe we want to push this down into the handlers 🤔 Anyhow, right now we can tell users to throw a `--verbose` on it to diagnose if they report issues (which we already do)

Reply:
Which is to say: you're right! Swallowing the error makes sense for now. Please make a quick tracking Issue and we can decide later 👍

Reply:
My thinking was this: If the version check fails, it's not so bad, so ignoring errors is fine.
If something else more critical needs the environment, it'll try to read the config file again anyway, and react to the errors accordingly.

Reply:
Right right 👍 Whiiiich is probably why it's the way it is. Got it 😅